OmsAgent/HandlerManifest.json

```diff
@@ -10,7 +10,9 @@
       "disableCommand": "omsagent_shim.sh -disable",
       "rebootAfterInstall": false,
       "reportHeartbeat": false,
-      "updateMode": "UpdateWithInstall"
+      "updateMode": "UpdateWithInstall",
+      "continueOnUpdateFailure": "true",
```

Comment:
I don't see this anywhere in the repo, where is it referenced?

Reply:
We never really created the PR for adding this to the Fairfax github code (forgot to add it during the whole Fairfax release mitigation). This feature is part of the newer waagent versions - https://microsoft.sharepoint.com/:w:/t/ComputeVM/EX85ZKjTWqNOoN7P5rZBRKgBRZ5cXR-TyTxlWttCi4ICOg?e=HU3Uhm 
This change is currently part of the 1.12.17 Fairfax extension's HandlerManifest.json as that is what allowed us to ignore the failed disable status and continue with newer version installation.

Reply:
Ok so it is a flag just for use by guest agent

Reply:
Yes, correct.

Reply:
Removed the "continueOnUpdateFailure": "true", flag from this PR. Will create a new PR for that change since it is out of scope for the general topic of this PR